demo/src/SvgDisplay.tsx

```diff
@@ -33,16 +34,31 @@ export default function SvgDisplay(props: SvgDisplayPropTypes) {
         <Paper
             sx={{
                 height: 500,
-                overflow: "auto",
+                overflow: "hidden",
+                display: "flex",
+                justifyContent: "center",
+                alignItems: "center",
             }}
             variant="outlined"
         >
-            <canvas
-                data-testid="memory-models-canvas"
-                ref={canvasRef}
-                width={canvasWidth}
-                height={canvasHeight}
-            />
+            <div style={{ height: 500, justifyContent: "center" }}>
```

Comment:
I'm not totally convinced you need a `div` here, but if you want to keep it you should make its `width` style `100%` to ensure it fills its parent